spec/services/candidate_interface/section_policy_spec.rb

```diff
@@ -45,12 +45,36 @@
       context 'when accessing an non editable section' do
         let(:controller_path) { 'some-non-editable/controller' }
 
-        it 'returns false' do
+        it 'returns true' do
```

Comment:
This is a bit confusing, what returns true? Typo?

Reply:
Typo, fixed!